acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/condition/net/NetConditions.java

```diff
@@ -59,4 +59,8 @@ public Condition netVersionExceptional(final String expectedMessage) {
   public Condition netVersionUnauthorized() {
     return new ExpectUnauthorized(transactions.netVersion());
   }
+
+  public Condition rpcIsListening() {
+    return new AwaitNetPeerCount(transactions.peerCount(), BigInteger.ZERO);
```

Comment:
expecting peerCount = 0 seems like we're adding a requirement that is not obvious from the name of the method `rpcIsListening` - we just want to check if we get an RPC response, right?